src/VirtoCommerce.Platform.Security/Handlers/LogChangesUserChangedEventHandler.cs

```diff
@@ -59,56 +61,19 @@ public virtual async Task Handle(UserResetPasswordEvent message)
             await SaveOperationLogAsync(message.UserId, "Password resets", EntryState.Modified);
         }
 
-        protected virtual ListDictionary<string, string> DetectAccountChanges(ApplicationUser newUser, ApplicationUser oldUser)
```

Comment:
Prevent breaking changes.

Reply:
there are no breaking changes, look at 
https://github.com/VirtoCommerce/vc-platform/blob/cb43edf93944e77854e85809167c1c586bb3a160/src/VirtoCommerce.Platform.Security/Handlers/LogChangesUserChangedEventHandler.cs#L74
it's just an issue with diff PR on UI